RELEASE.md

```diff
@@ -30,6 +30,14 @@ Keras pruning API:
 * Tested against TensorFlow 1.14.0, 2.0.0, and nightly, and Python 3.
 
 
+# TensorFlow Model Optimization next release TBD
+
+Keras clustering API:
+
+* Added *ClusteringSummaries* to create additional output for the clustering 
+progress for TensorBoard.
+* Tested against TensorFlow 1.14.0, 2.0.0, and nightly, and Python 3.
```

Comment:
nit: 2.3.0, in 0.5.0 we are talking about the latest versions

Reply:
Actually 2.0.0 is fine and intended. 

The idea is that if it passes against 2.0.0 and nightly, then it should generally pass on all versions in between (including 2.1 / 2.2) given that 2.X has no backward compatible changes. It won't be perfect (e.g. miscellaneous bugs), but it's the most practical choice given we cannot test across all versions.

Reply:
And until we communicate otherwise, currently TFMOT is expected to be compatible with 2.0+.

Reply:
Thanks for the clarification, Alan!